lib/tokenize.cpp

```diff
@@ -1544,13 +1544,14 @@ void Tokenizer::simplifyTypedef()
                                     tok2 = tok2->tokAt(2);
 
                                 // skip over function parameters
-                                if (tok2->strAt(1) == "(") {
+                                if (tok2->str() == "(" || tok2->strAt(1) == "(") {
```

Comment:
if `tok2->str() == "("` is true.. then don't you want to jump to the ")" on the next line?

Reply:
Yes indeed. Thanks.